src/scib_metrics/utils/_pca.py

```diff
@@ -0,0 +1,159 @@
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+import jax
+import jax.numpy as jnp
+
+from .._types import NdArray
+from ._utils import get_ndarray
+
+
+@dataclass
+class _SVD:
```

Comment:
just to avoid conflicting with SVD running the algorithm itself, I would call this SVDOutput or SVDResult